GitUtils.ps1

```diff
@@ -94,14 +162,30 @@ function Get-GitBranch($gitDir = $(Get-GitDirectory), [Diagnostics.Stopwatch]$sw
     }
 }
 
-function Get-GitStatus($gitDir = (Get-GitDirectory)) {
+<#
+.SYNOPSIS
+    Show the working tree status.
+#>
+function Get-GitStatus {
+    param(
+        [parameter(ValueFromPipeline=$true,Position=0)]
+        [string[]] $GitDir = (Get-GitDirectory)
+    )
+    process {
+        $GitDir | foreach {
+            return GitStatusOf($_)
+        }
+    }
+}
+
+function GitStatusOf($GitDir) {
     $settings = $Global:GitPromptSettings
     $enabled = (-not $settings) -or $settings.EnablePromptStatus

```

Comment:
Maybe YAGNI, but it might be reasonable to add a `-Force` parameter to override `$GitPromptSettings.EnablePromptStatus`.